libs/execution_context/unit/execution_context_unit_test.cpp

```diff
@@ -178,6 +182,9 @@ _test_crud_operations(ebpf_map_type_t map_type)
         dpc = {emulate_dpc_t(1)};
     }
 
+    // For each map type, maximum only one of replace_on_full and insert_on_full should be true.
```

Comment:
```suggestion
    // For each map type, at most only one of replace_on_full and insert_on_full should be true.
```
But it would be better to remove this constraint and replace the two bools with one 3-valued enum.